connector/avro/src/main/scala/org/apache/spark/sql/avro/AvroUtils.scala

```diff
@@ -50,8 +50,8 @@ private[sql] object AvroUtils extends Logging {
     val conf = spark.sessionState.newHadoopConfWithOptions(options)
     val parsedOptions = new AvroOptions(options, conf)
 
-    if (parsedOptions.parameters.contains(ignoreExtensionKey)) {
-      logWarning(s"Option $ignoreExtensionKey is deprecated. Please use the " +
+    if (parsedOptions.parameters.contains(IGNORE_EXTENSION_KEY.toString)) {
```

Comment:
this is what I don't like about enums. You need to go and call `toString` everywhere

Reply:
I see, that's a reasonable pain point. I will move away from Enum then.